__tests__/ros-ci.test.ts

```diff
@@ -95,16 +95,16 @@ Blah blah.
 Description of the changes.
 Blah blah.
 
-action-ros-ci-repos-override:   
-action-ros-ci-repos-override: https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
+action-ros-ci-repos-override:
+action-ros-ci-repos-override: https://raw.githubusercontent.com/ros2/ros2/rolling/ros2.repos
 action-ros-ci-repos-override : https://some.website.repos
  action-ros-ci-repos-override:  https://gist.github.com/some-user/some-gist.repos
  action-ros-ci-repos-supplemental:https://gist.github.com/some-user/some-other-gist.repos
-action-ros-ci-repos-supplemental:  file://path/to/some/file.txt 
+action-ros-ci-repos-supplemental:  file://path/to/some/file.txt
```

Comment:
It's not a big deal, but could you revert the whitespace changes you made (probably made by your IDE?)? They were intentional.